packages/cocoindex/tests/test_proxy_manager.py

```
@@ -613,6 +613,29 @@ def test_skips_when_ready(
         mock_popen.assert_not_called()
 
+    @patch("proxy_manager.subprocess.Popen")
+    @patch("proxy_manager.get_proxy_state", return_value={"proxy_state": "stopped"})
+    def test_does_not_block_on_helper_process(
+        self,
+        mock_state: MagicMock,
+        mock_popen: MagicMock,
+        tmp_path: Path,
+    ) -> None:
+        """EV-18: warmup はバックグラウンド起動後、helper の終了を待たずに返る
+        （SessionStart hook が同期的にブロックしないことの根拠。Issue #127 仕様確定）。
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)